tools/export-analyzer/main.go

```diff
@@ -50,43 +61,63 @@ func main() {
 	}
 
 	// Do some basic data validation.
-	if !*quiet {
-		checkExportFile(keyExport)
+	success := true
+	if err := checkExportFile(keyExport); err != nil {
+		success = false
+		if !*quiet {
+			log.Printf("export file contains errors: %v", err)
+		}
 	}
 
-	prettyJSON, err := json.MarshalIndent(keyExport, "", "  ")
-	if err != nil {
-		log.Fatalf("error pretty printing export: %v", err)
+	if *printJSON {
+		prettyJSON, err := json.MarshalIndent(keyExport, "", "  ")
+		if err != nil {
+			log.Fatalf("error pretty printing export: %v", err)
+		}
+		log.Printf("%v", string(prettyJSON))
```

Comment:
If this is intended to be here, can we prefix with "output" or something. If it's debug, let's remove it.

Reply:
yes, that was the original reason for this tool to print the export file contents